config/cscs.py

```diff
@@ -130,7 +130,9 @@ class ReframeSettings:
                                      'PrgEnv-cray_aj_b'],
                         'descr': 'Kesch compute nodes',
                         'resources': {
-                            '_rfm_gpu': ['--gres=gpu:{num_gpus_per_node}']
+                            '_rfm_gpu': ['--gres=gpu:{num_gpus_per_node}'],
+                            'distribution': ['--distribution=block:block'],
+                            'cpu_bind' : ['--cpu_bind=q']
```

Comment:
The resource names must be more generic and allow tests to pass options:

```python
'resources': {
    'task_placement': ['--distribution={distribution}', '--cpu_bind={cpu_binding}']
}
```

Then you should define the `extra_resources` as follows:

```python
self.extra_resources = {
    'task_placement': {
        'distribution': 'block:block',
        'cpu_binding': 'q'
}
```

But I don't think that the `--cpu_bind` option is needed: `q` is the default value.

Reply:
this comment is outdated

Reply:
Why this comment is outdated?